base/strings/basic.jl

```diff
@@ -79,6 +79,18 @@ Return the code unit value in the string `s` at index `i`. Note that
 I.e. the value returned by `codeunit(s, i)` is of the type returned by
 `codeunit(s)`.
 
+# Examples
+```jldoctest
+julia> codeunit("Hello", 1)
+0x48
+
+julia> codeunit("Hello", 2)
+0x65
+
+julia> typeof(codeunit("Hello", 2))
```

Comment:
Could save the value in a variable and call typeof on that instead, which might make this line simpler.